_posts/2019-12-13-snowplow-unity-tracker-0.4.0-released/index.md

```diff
@@ -0,0 +1,100 @@
+
+Read on below the fold for:
+
+1. [Updates to .NET Standard 2.0](#update-netstandard)
+2. [Improvements to Event Storage](#event-storage-improvements)
+3. [New Demo Unity Game](#new-demo-app)
+4. [Updates and bug fixes](#updates)
+5. [Upgrading](#upgrade)
+6. [Documentation and help](#doc)
+
+<!--more-->
+
+<h2 id="update-netstandard">1. Updates to .NET Standard 2.0</h2>
+The release of Unity 2018.1 and beyond brought support for the new Stable Scripting Runtime which upgrades .NET support to .NET 4.x and .NET Standard 2.0 profiles. This new feature allows us to move to new frameworks, modern .NET libraries and a variety of improvements to our Tracker codebase by upgrading to a .NET Standard 2.0 library.
+
+During this upgrade we have also placed considerable effort in ensuring our Tracker is capable of working on popular Unity platforms, both Mono and IL2CPP, including: Windows, macOS, Linux, iOS and Android. The Unity Tracker may also be compatible with other Unity platforms but they have not been tested as part of this release.
+
+Our Test Framework has also been updated to run as a .NET Standard 2.0 Unity 2018.4 Project, allowing for integration testing of our Snowplow Tracker within the Unity environment. You can find our how to run the updated Test Suite on the Unity Tracker [README][readme-testing].
+
+<h2 id="event-storage-improvements">2. Improvements to Event Storage</h2>
+By moving to .NET Standard 2.0, this has allowed us to leverage a new storage technology that is 100% managed C# code, with no native library dependencies, called LiteDB. By doing so, we have been able to fix compatibility issues running the Unity Tracker on some Android devices (in particular on arm64 devices).
+
+This change requires new dlls to be deployed along with version 0.4.0 of the Unity Tracker however they will automatically be included when importing the SnowplowTracker.unitypackage from our GitHub release. We advise dlls imported from previous Unity Tracker releases are deleted before importing version 0.4.0.
+
+Additionally, we have also exposed a new IStore interface when initialising the Emitter object. This will allow users of the Unity Tracker to either alter the construction of the EventStore object, as shown below, or by creating an implementation of the IStore interface that fits the needs of the application.
+
+{% highlight csharp %}
+var emitter = new AsyncEmitter(collectorUrl, HttpProtocol.HTTPS, HttpMethod.POST, 1, 52000L, 52000L, new EventStore("filename.db", true));
+{% endhighlight %}
+
+<h2 id="new-demo-app">3. New Demo Unity Game</h2>
+We've also created a new Demo Game that has been built with Unity 2018.4. This has allowed the new .NET Standard version of the Snowplow Unity Tracker to be included within the Demo Game as well as some aspects of the Demo game, such as the User Interface, to use the latest Unity features.
```

Comment:
I think I know what you're saying here, but it's a bit confusing. Using Unity 2018.4 has allows the .Net Standard version to be used, not creating the Demo game itself? The second sentence is a confusing read too. 

Minor point, but I'd use all lower case for demo game, but whatever you decide on keep it consistent.